LocationExport/Program.cs

```diff
@@ -27,8 +27,8 @@ static void Main(string[] args)
                 return;
             }
 
-            _reader1 = new TR1LevelReader();
-            _reader3 = new TR3LevelReader();
+            _reader1 = new TR1LevelControl();
+            _reader3 = new TR3LevelControl();
```

Comment:
Isn't it enough to call `new()`?

Reply:
Yes, I missed quite a few of these, so updated now.

We have lots of messages like this since migrating to .NET Core, I should aim to fix these while editing the affected files. Ideally this number will eventually become 0.

![image](https://github.com/LostArtefacts/TR-Rando/assets/33758420/7797a99e-c18b-4ff2-b898-d53403622af9)

Reply:
Maybe this could be tackled soon so that the PRs / CRs don't get polluted with stylistic comments?

Reply:
Thanks, I'll organise some cleanup PRs for this.